charts/kubewarden-controller/templates/webhooks.yaml

```diff
@@ -1,13 +1,18 @@
-# generate certificates
-{{ $dnsName :=  printf "%s-webhook-service.%s.svc" (include "kubewarden-controller.fullname" .) .Release.Namespace }}
+# generate CA
 {{ $ca := genCAWithKey "kubewarden-controller-ca" 3650 (genPrivateKey "ecdsa") }}
-{{ $cert := genSignedCertWithKey $dnsName nil ( list $dnsName ) 365 $ca (genPrivateKey "ecdsa") }}
 {{ $caCert := ($ca.Cert | b64enc) }}
 {{ $oldCaCert := "" }}
 {{ $caBundle := $caCert }}
 {{ $caPrivateKey := ($ca.Key | b64enc) }}
-{{ $serverCert := ($cert.Cert | b64enc) }}
-{{ $serverPrivateKey := ($cert.Key | b64enc) }}
+# generate webhook server certificates
+{{ $controllerDNSName :=  printf "%s-webhook-service.%s.svc" (include "kubewarden-controller.fullname" .) .Release.Namespace }}
+{{ $webhookServerCertKeyPair := genSignedCertWithKey $controllerDNSName nil ( list $controllerDNSName ) 365 $ca (genPrivateKey "ecdsa") }}
+{{ $webhookServerCert := ($webhookServerCertKeyPair.Cert | b64enc) }}
+{{ $webhookServerKey := ($webhookServerCertKeyPair.Key | b64enc) }}
+# generate audit-scanner client certificates
+{{ $auditScannerClientCertKeyPair := genSignedCertWithKey "audit-scanner-client" nil nil 365 $ca (genPrivateKey "ecdsa") }}
+{{ $auditScannerClientCert := ($auditScannerClientCertKeyPair.Cert | b64enc) }}
+{{ $auditScannerClientKey := ($auditScannerClientCertKeyPair.Key | b64enc) }}
```

Comment:
Just a reminder for all of us, I think we need to update ArgoCD installation instructions to ignore  the audit scanner secret as well